src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="content">
+    <div class="calendar calendar--start-day--thu calendar--month-length--31">
```

Comment:
start day thursday, but fist day on first column (should be on 4)
 https://prnt.sc/tvQq81M8dZM2